docs/introduction_tutorial/intro_tutorial_part2.ipynb

```diff
@@ -0,0 +1,1228 @@
+{
```

Comment:
<p>Before we get here, I think we need a quick note about how to select a recipe class, or at least a link to the relevant docs (see #298).</p><p> </p><p>It's okay to leave at as is for now, but we will need to remember to update here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-forge/pangeo-forge-recipes/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>